packages/storage/amplify_storage_s3/example/lib/main.dart

```diff
@@ -105,19 +108,271 @@ class HomeScreen extends StatefulWidget {
 }
 
 class _HomeScreenState extends State<HomeScreen> {
+  var list = [];
+  var imageUrl = '';
+
+  @override
+  void initState() {
+    super.initState();
+    _checkAuthStatus();
+    _listAllPublicFiles();
+  }
+
+  // sign out of the app
+  Future<void> _signOut() async {
+    try {
+      await Amplify.Auth.signOut();
+      _logger.debug('Signed out');
+    } on AuthException catch (e) {
+      _logger.debug('Could not sign out - ${e.message}');
```

Comment:
Error statements should be logged at error level:

```suggestion
      _logger.error('Could not sign out - ${e.message}');
```

Reply:
And you can pass the full error as the second arg:

```suggestion
      _logger.error('Could not sign out', e);
```